data/free worlds intro.txt

```diff
@@ -546,6 +572,26 @@ mission "FW Katya 1"
 	on decline
 		event "FW Katya Alt 2: ready" 21
 
+mission "logbook: karate chop decline"
+	invisible
+	to offer
+		has "karate chop"
+		has "FW Katya 1: offered"
+	on offer
+		log `I karate chopped a militia officer who was holding a gun to the back of my head. Luckily the gun wasn't loaded, but the karate chop didn't work as expected. (I knew I should have studied karate while on New Boston...)`
```

Comment:
"karate chop" appears a bit too often in this.

`(I knew I should have studied karate while on New Boston...)`
"while on" feels weird. Try something like "back on".